### PR TITLE
Multi arch build for piraeus-client

### PR DESCRIPTION
**Pull request by toelke**
This is a (improved) copy of #76 as the original author seems MIA.
The issue is #74

### CODE REVIEW

**Review by WanzenBug — CHANGES_REQUESTED** [review]

File: .github/workflows/build-piraeus-client.yaml [hunk 1]
```diff
@@ -11,11 +11,43 @@ jobs:
...
-      - uses: actions/checkout@v2
-      - name: build images
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Prepare
+        id: prep
         run: |
+          DOCKERHUB_IMAGE=docker.io/${{ secrets.DOCKERHUB_USERNAME }}/piraeus-client
+          QUAYIO_IMAGE=quay.io/${{ secrets.QUAYIO_USERNAME }}/piraeus-client
```
[the remark this answers is not in the document]
Comment (toelke) [reply .github/workflows/build-piraeus-client.yaml]: Yes, that sounds reasonable. Thanks for catching that.

**Review by WanzenBug — CHANGES_REQUESTED** [review]
Looks good!

One thing that I'd add, not directly related to the issue: run the workflow on pull requests:
```
on:
  pull_requests:
    paths: 
      - ...
```

and then just disable the push parameter in the last step:

```
with:
  push: ${{ github.event_name != "pull_request" }}
```
(might need some tweaks, I haven't tested it yet). You then might also need to disable the login step for pull_request jobs.

But that would definitly help with verifying these changes.

Oh, and once you are done, please squash all commits into a single one, the change is small enough to be just a single commit.

**Comment by toelke** [comment]
I have not used github actions before, but I need to learn it anyways :-D

**Comment by WanzenBug** [comment]
> I have not used github actions before, but I need to learn it anyways :-D

Thanks for giving it a try! 

I think I made a wrong suggestion, it should be `pull_request` (note the singular).

**Comment by toelke** [comment]
I caught that from the error message :-D

I do not feel comfortable to squash all of this, as the first commit is by someone else (see #74 ) -- should I squash to two commits?

**Comment by WanzenBug** [comment]
Two commits is also fine.

**Review by WanzenBug — APPROVED** [review]

**Comment by WanzenBug** [comment]
LGTM!

Thank you @toelke and @Jukie for your contribution! :tada:

**Comment by toelke** [comment]
Thank you for your help!

**Pull request merged**